site/data/apiVersions.yaml

```diff
@@ -1 +1 @@
-- v1alpha3
+- v1alpha4
```

Comment:
https://github.com/kubernetes-sigs/kind/blob/b6bc112522651d98c81823df56b7afa511459a3b/site/layouts/index.redirects#L1-L3

looks like both versions should be added here, not only one of them?

Reply:
I don’t know if we still need v1alpha3,  The [master branch](https://github.com/kubernetes-sigs/kind/tree/main/pkg/apis/config) only has v1alpha4. and the [latest version](https://pkg.go.dev/sigs.k8s.io/kind/pkg/apis/config/v1alpha3?utm_source=godoc) of v1alpha3 was published on May 1, 2020

Reply:
/lgtm